clustered/server/src/main/java/org/ehcache/clustered/server/ClusterTierManagerPassiveEntity.java

```diff
@@ -87,31 +60,11 @@ public ClusterTierManagerPassiveEntity(ClusterTierManagerConfiguration config,
     }
   }
 
-  private void invokeRetirementMessages(PassiveReplicationMessage message) throws ClusterException {
-
-    switch (message.getMessageType()) {
-      case CLIENT_ID_TRACK_OP:
-        ehcacheStateService.getClientMessageTracker().remove(message.getClientId());
-        break;
-      default:
-        throw new AssertionError("Unsupported Retirement Message : " + message);
-    }
-  }
-
-  private void invokeLifeCycleOperation(LifecycleMessage message) throws ClusterException {
-    switch (message.getMessageType()) {
-      case VALIDATE:
-        applyMessage(message);
-        break;
-      default:
-        throw new AssertionError("Unsupported LifeCycle operation " + message.getMessageType());
-    }
+  @Override
+  public void invoke(EhcacheEntityMessage message) {
+    throw new AssertionError("No invokes supported by this entity");
```

Comment:
I kept this here just to make sure that no surprise invocations are coming to this entity.